CultivateMVP/JournalOptionsScene/JournalOptionsViewModel.swift

```diff
@@ -1,17 +1,52 @@
 import Foundation
 
 protocol JournalOptionsViewModelType {
-    var entries: [Entry] { get }
-    var moodEntry: MoodEntry { get }
+    // to do - turn into plain UI things
+    var entries: [Entry]? { get }
+    var moodEntry: MoodEntry? { get }
```

Comment:
Yes and should be a `Driver`.  Describe one `JournalOptionsViewState` and put all plain thing there + here should be one output `Driver<JournalOptionsViewState >`